doc/extdev/parserapi.rst

```diff
@@ -3,6 +3,35 @@
 Parser API
 ==========
 
+`The document of docutils describes`__ about the parsers like following:
```

Comment:
The docutils documentation describes parsers as follows:

Reply:
done